stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/components/avatar/ChannelAvatar.kt

```diff
@@ -53,7 +56,12 @@ public fun ChannelAvatar(
          * If the channel has an image we load that as a priority.
          */
         channel.image.isNotEmpty() -> {
-            val painter = rememberImagePainter(data = channel.image)
+            val painter = if (LocalInspectionMode.current) {
+                // Show hardcoded avatar from resources when rendering preview
+                painterResource(id = R.drawable.stream_compose_preview_avatar)
```

Comment:
Thinking about introducing one more hardcoded image resource here that represent a channel with image.

Reply:
The issue with this here is that this check only works if the `channel.image` property is not empty. We could do that (potentially) by pointing to a local asset or something, or just by faking the image to be non empty and then using the said resource.

Either way, it works for me, and if it's not much work, feel free to jump on it!